src/main/java/use_cases/GetInfo/GetInfoInteractor.java

```diff
@@ -0,0 +1,41 @@
+package use_cases.GetInfo;
+
+import entities.CompanyInformation;
+import use_cases.APIAccessInterface;
+
+public class GetInfoInteractor implements GetInfoInputBoundary {
+    GetInfoOutputBoundary getInfoPresenter;
+    APIAccessInterface driverAPI;
+
+    public GetInfoInteractor(GetInfoOutputBoundary getInfoPresenter, APIAccessInterface driverAPI) {
+        this.getInfoPresenter = getInfoPresenter;
+        this.driverAPI = driverAPI;
+    }
+
+    @Override
+    public void execute(GetInfoInputData getInfoInputData) {
+        /**
+         * The getInfoInputData parameter should follow the specifications laid out in that class.
+         * <p>
+         * This method implements the bulk of the GetInfo use case.
+         * Info is fetched using the Finnhub driver.
+         *
+         * @param  getInfoInputData  an InputData object following the relevant CA Engine rules
+         */
```

Comment:
I feel that the documentation needs to mention what the use case does, or rather, what the execute method does. currently, it just says it does the bulk of the use case, what does that mean?

Also, I don't think you need to mention it follows CA engine rules, the client doesn't really care about that. Instead, mention that the InputData contains the ticker